artnet/__init__.py

```diff
@@ -60,6 +166,17 @@ def __call__(self, var1=None, var2=None):
         if type(var1) == list and type(var2) == type(None):
             self.send_frame(var1)
 
+    def _update_loop(self):
+        if not self.alive:
+            return
+        self.logger.debug("Update Loop")
```

Comment:
Currently the update method is empty. So in case someone configured a update cycle nothing will happen here.

Reply:
Ooops, will do that asap.